data/drak/drak ships.txt

```diff
@@ -50,6 +50,8 @@ ship "Archon"
 		"turn" 2400
 		"turning energy" 10
 		"turning heat" 5
+		"outfit scan power" 250
+		"outfit scan efficiency" 150
```

Comment:
I think the cloaked variant is the one you need to modify, unless you want _all_ Archons to scan the player. Also, remember it needs `"cloaked scanning"`, too.

Reply:
Initially I thought all Archons _should_ probably be able to scan, but it probably does make sense if it's just the cloaked variant. Swapped that and added `"cloaked scanning"` (so many neat new attributes with 10.7)